.travis.yml

```diff
@@ -16,12 +19,14 @@ before_install:
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
 
 before_script:
-  - npm install -g jpm mozilla-download
+  - npm install -g jpm
```

Comment:
Not even 💯 sure we need to globally install `jpm` here since we already have it installed as a devDependency, per [./package.json:25](https://github.com/mozilla-services/pageshot/blob/f096400e270901679450bda07f33ae9d51a3dfd9/package.json#L25).

We can probably just reference that one via `./node_modules/.bin/jpm` path, or if we refer to it in a package.json script, it will automatically prepend `./node_modules/bin/` to our `$PATH` env variable for us.

Similarly, we can probably remove the `npm install` below on .travis.yml:23 since the Travis-CI `install` hook should have run before the `before_script` hook. You can see this in action by looking at the latest Travis-CI logs:

```sh
$ npm install        # 212.85s
$ npm install -g jpm #  23.86s
$ npm install        #   7.66s
```

So as you can see, installing `jpm` a second time, adds ~24s to the build, then running `npm install` again adds another ~8s (but doesn't add any additional modules since we just finished doing the `npm install` on line 165 of the Travis-CI log).

Builds seem to take 12-15 minutes to complete though, so +/-30 seconds isn't a huge impact, but still. 🕐